attest/tpm_linux.go

```diff
@@ -140,7 +148,11 @@ func (t *platformTPM) close() error {
 	case TPMVersion12:
 		return t.ctx.Close()
 	case TPMVersion20:
-		return t.rwc.Close()
+		rwc, ok := t.rw.(io.ReadWriteCloser)
```

Comment:
Please revert the s/io.ReadWriteCloser/io.ReadWriter/g changes. For the CommandChallen you can make Close no-op:

```
type noOpCloser struct {
    io.ReadWriter
}
func (n noOpCloser) Close() error {
    return
}
```